src/System.Net.Http.WinHttpHandler/System.Net.Http.WinHttpHandler.sln

```diff
@@ -31,10 +31,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.ActiveCfg = netstandard-Windows_NT-Debug|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.Build.0 = netstandard-Windows_NT-Debug|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Release|Any CPU.ActiveCfg = netstandard-Windows_NT-Release|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Release|Any CPU.Build.0 = netstandard-Windows_NT-Release|Any CPU
+		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.ActiveCfg = netcoreapp-Windows_NT-Debug|Any CPU
```

Comment:
Aren't these 'netcoreapp-Windows_NT-*' duplicates of the ones below....the only difference being the GUIDs are different?  Why have two of the same thing now?

Reply:
Isn't that the test project? I did use the `UpdateVSConfigurations` target to produce these changes.

Reply:
Yes, you're right.  It is the test projects.